compiler/ml/ast_untagged_variants.ml

```diff
@@ -89,6 +100,16 @@ type tag = {name: string; tag_type: tag_type option}
 type block = {tag: tag; tag_name: string option; block_type: block_type option}
 type switch_names = {consts: tag array; blocks: block array}
 
+let tag_type_to_type_string = function
```

Comment:
Nit: maybe just call it `tag_type_to_string` (similar to `block_type_to_string`)?